tools/buildmgr/cbuildgen/scripts/cbuild.sh

```diff
@@ -323,7 +323,7 @@ fi
 if [ -e "${abs_intdir}${project}.cpinstall" ]
   then
   dos2unix "${abs_intdir}${project}.cpinstall" >&${out} 2>&${err}
-  cp_install.sh "${abs_intdir}${project}.cpinstall" >&${out} 2>&${err}
+  cpackget pack add --agree-embedded-license --packs-list-filename "${abs_intdir}${project}.cpinstall" >&${out} 2>&${err}
```

Comment:
I would suggest to add the `-v` option for printing info messages, otherwise the cpackget call is completely silent.
```suggestion
  cpackget pack add -v --agree-embedded-license --packs-list-filename "${abs_intdir}${project}.cpinstall" >&${out} 2>&${err}
```